darwin/dataset/remote_dataset.py

```diff
@@ -273,9 +269,20 @@ def release(self, name: Optional[str] = None):
         release: Release
             The release created right now
         """
-        self.client.post(f"/datasets/{self.dataset_id}/exports", team=self.team)
+        if annotation_class_ids is None:
+            annotation_class_ids = []
+        payload = {"annotation_class_ids": annotation_class_ids, "name": name}
+        self.client.post(f"/datasets/{self.dataset_id}/exports", payload=payload, team=self.team)
```

Comment:
should probably use `error_handlers=[name_taken, validation_error]`

Reply:
we should also add an error handler for `{'errors': {'name': ['is reserved']}}`

Reply:
I added the `error_handlers=[name_taken, validation_error]` but I am not entirely certain what you mean by the `{'errors': {'name': ['is reserved']}}` (respectively, I am not sure how to address it?)